src/logo-system/logo-elements/index.md

```diff
@@ -49,11 +49,11 @@ This ensures visual balance and harmony between the elements.
 
 The pixel sizes displayed on this page are indicative and will vary across devices.
 
-<div>
+{% sectionHighlight { classes: 'app-section-highlight__wrapper--logo govuk-!-margin-top-6 govuk-!-margin-bottom-6' } %}
```

Comment:
Not a review comment but a note to self to go back and scrub these out in https://github.com/alphagov/govuk-brand-guidelines/pull/176 once this is merged and that one can be worked on with more authority